### PR TITLE
Build an oapp

### PR DESCRIPTION
**Pull request by alijnmerchant21**
This PR is aimed to demonstrate how to deploy a simple Hello World smart contract using cosmwasm. 

## Summary by CodeRabbit

- **Documentation**
	- Enhanced the guide for creating and deploying a "Hello World" smart contract, providing detailed instructions and prerequisites.
	- Improved clarity and accessibility of the WardenJS documentation with a concise introduction and navigation adjustments.
	- Streamlined the introduction to Omnichain Applications (OApps) and reorganized content for better readability.
	- Adjusted sidebar positions for various documents to improve navigation structure.
	- Removed a dropdown menu in the documentation configuration for a cleaner sidebar layout.

### CODE REVIEW

**Review by ijonele — DISMISSED** [review]

**Review by jjheywood — APPROVED** [review]

File: docs/developer-docs/docs/build-an-app/build-a-custom-smart-contract.md [hunk 1]
```diff
@@ -1,13 +1,221 @@
...
+
+   ```bash
+   rustup target add wasm32-unknown-unknown
+   cargo install cargo-generate --features vendored-openssl
+   brew install binaryen
+   ```
+
+- [Run a local chain](test/run-a-local-chain) and make sure you have `wardend` correctly installed. You can stop the chain for now if you wish.
+
+   The next steps require your local account name, or key name, which is referenced as `my-key-name` in the provided command-line examples. You can check the list of available keys by executing this command (while the node is running):
```
Comment (pitasi) [inline docs/developer-docs/docs/build-an-app/build-a-custom-smart-contract.md:39]: it doesn't require the node to be running

File: docs/developer-docs/docs/build-an-app/build-a-custom-smart-contract.md [hunk 2]
```diff
@@ -1,13 +1,221 @@
...
+   
+   To install these tools, run the following commands:
+
+   ```bash
+   rustup target add wasm32-unknown-unknown
+   cargo install cargo-generate --features vendored-openssl
+   brew install binaryen
+   ```
+
+- [Run a local chain](test/run-a-local-chain) and make sure you have `wardend` correctly installed. You can stop the chain for now if you wish.
```
Comment (pitasi) [inline docs/developer-docs/docs/build-an-app/build-a-custom-smart-contract.md:37]: > You can stop the chain for now if you wish.

I'd rather either explain the benefit of stopping the chain, or avoid suggesting it

Comment (alijnmerchant21) [reply docs/developer-docs/docs/build-an-app/build-a-custom-smart-contract.md:37]: @ijonele please take note.

File: docs/developer-docs/docs/build-an-app/build-a-custom-smart-contract.md (hunk 1 above)
Comment (alijnmerchant21) [reply docs/developer-docs/docs/build-an-app/build-a-custom-smart-contract.md:39]: > it doesn't require the node to be running

The chain needs to be running only while deploying a contract. We can either do it at the beginning or middle of the tutorial. I think in the start looks clean. But I'm open to moving it to later part of the tutorial too.

File: docs/developer-docs/docs/build-an-app/build-a-custom-smart-contract.md (hunk 2 above)
Comment (ijonele) [reply docs/developer-docs/docs/build-an-app/build-a-custom-smart-contract.md:37]: But In Step 6 the node has to be running? I guess in this case we can just say - Run a local chain and not mention stopping it.

**Pull request merged**